NavigationBasicSample/README.md

```diff
@@ -11,6 +11,8 @@ This sample showcases the following features of the Navigation component:
  * Arguments (profile screen receives a user name)
  * Deep links (`www.example.com/user/{user name}` opens the profile screen)
 
+### Screenshots
```

Comment:
Screenshots is plural, but you only have one screenshot?

Reply:
Keeping it consistent with other samples like https://github.com/googlesamples/android-EmojiCompat/#screenshots
(required for preview in Android Studio via Import Sample)

Reply:
SGTM